packages/sapui5-downloader-core/src/index.js

```diff
@@ -100,6 +100,8 @@ class Downloader {
 
     const sapui5DownloadURL = url.resolve(this.downloadEndpoint.href, `sapui5-${this.type}-${sapui5Version}.zip`)
 
+    console.log(`SAPUI5 download URL: ${sapui5DownloadURL}`)
+
```

Comment:
Could we combine this with the console log in line 109? e.g:
```javascript
console.log(`Downloading SAPUI5 from ${sapui5DownloadURL}...`)
```

Reply:
Done